### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by Seeeeler** [interaapps/pastefy#9] [General]
Hello,

How can I install pastefy and use it? Can you please provide detailed installation guide?
Also, can you add captcha protection support?

**Comment by JulianFun123** [comment]
Hello Seeeeler,
we have a page about self-hosting on the pastefy wiki on github. 
You might use Docker or Docker Compose.
https://github.com/interaapps/pastefy/wiki/Installation.
There is no support for captcha.

**Reply by Seeeeler** [reply]
Got it, thank you

Features request:
Can we have captcha (google captcha) protection for the pastes?
I see that shorten link after creating a paste is automatically copied but can we have short link of the paste somewhere on the page other than browser's address bar?

**Reply by JulianFun123** [reply]
A copy url button or something like this would just waste space because you can just look into the url-bar of your browser.
